Etf 2.1

jbarreroj opened the pull request:
Updated the compilation and building instructions in the documentation according to our processes.

manmorjim committed:
Adding CODE_OF_CONDUCT.adoc (draft version)

Adding the draft version of the Code of Conduct in AsciiDoc format. The contact email is pending.

jenriquesoriano committed:
Merge branch 'etf-validator:master' into master

manuelaromero-guadaltel committed:
added v2.1 and modified admin manual

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

jenriquesoriano committed:
Update Jetty9_Linux.adoc

manuelaromero-guadaltel committed:
modified admin manuals of version 2.1

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

manuelaromero-guadaltel committed:
created new index.html of admin in v2.1

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

manuelaromero-guadaltel committed:
modified index and added images to admin manual

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

jenriquesoriano committed:
Update Jetty9_Linux.adoc

Typo corrected

jenriquesoriano committed:
Update Jetty9_Windows.adoc

manuelaromero-guadaltel committed:
Updated docs to v2.1

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

manuelaromero-guadaltel committed:
Updated docs to v2.1

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

manuelaromero-guadaltel committed:
Updated docs to v2.1

Signed-off-by: manuelaromero-guadaltel <manuelaromero@guadaltel.es>

jonherrmann commented on v2.1/Admin_manuals/Jetty9_Linux.adoc:
```diff
@@ -0,0 +1,48 @@
+
+Inside the ETF-webapp create a testdrivers folder, in which the  other three resources (etf-suitd-legacy-2.1.0-SNAPSHOT.jar, etf-tetd-1.1.0-SNAPSHOT.jar, etf-bsxtd-legacy-2.1.0-SNAPSHOT.jar) have to be added. 
+
+Download and include the following libraries inside etf-webapp.war/WEB-INF/lib/ to make the deployment work properly:
```
As discussed in the last SG meeting this is a workaround that we should not publish in the documentation. Where should the administrator get the libraries from and is she/he really responsible for updating in case of security issues in these libraries...

Please add the libraries as dependencies to the SoapUI test driver. If you need assistance I am happy to help.

jonherrmann commented:
Thank you @jenriquesoriano and @jbarreroj  for the pull request, I appreciate the work you put into updating the documentation.

I spotted the part where the libraries for SoapUI have to be downloaded and put into the Jetty installation directory. From my experience, this steps will be rejected by a lot of Administrators, so I would like to ask you to update this part.

If you need assistance I am happy to help.

jonherrmann commented:
Dear @jenriquesoriano and @jbarreroj,

you could have searched maven central for the names of the dependencies, e.g.

https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on/1.71

and then entered them in the SoapUI test driver.

I have applied this and the tests now pass in the GitHub pipelines. The text would have to be removed accordingly.

jonherrmann commented:
Xmx parameter needs to be documented. See https://github.com/etf-validator/etf-webapp/issues/166

jenriquesoriano committed:
Removed the inclusion of additional libraries

Removed the inclusion of additional libraries as they have been added by JH in the pipeline

jenriquesoriano committed:
Removed the inclusion of additional libraries

Removed the inclusion of additional libraries as they have been added by JH in the pipeline

jenriquesoriano committed:
Xmx4GB reccommendation according to #166 for Linux

Updated instructions to set -Xmx to 4GB at least according to https://github.com/etf-validator/etf-webapp/issues/166

jenriquesoriano committed:
Xmx4GB reccommendation according to #166 for Windows

Updated instructions to set -Xmx to 4GB at least according to etf-validator/etf-webapp#166

jonherrmann commented:
Perfect 🙏
